### PR TITLE
PUBDEV-8869: Added optional `matplotlib` requirement for plotting in python

### PR DESCRIPTION
**Pull request by hannah-tillman**
For: [PUBDEV-8869](https://h2oai.atlassian.net/browse/PUBDEV-8869)

### CODE REVIEW

File: h2o-dist/index.html [hunk 1]
```diff
@@ -545,7 +545,9 @@ <h2>Use H<sub>2</sub>O directly from Python</h2>
                 <p class="terminal" id="to_copy8">
                     pip install requests<br/>
                     pip install tabulate<br/>
-                    pip install future
+                    pip install future<br/>
```
Comment (michalkurka) [inline h2o-dist/index.html:548]: Does the copy-to-clipboard feature still work?

Should it instead be 2 cells? Since the second one is optional?

Comment (ledell) [reply h2o-dist/index.html:548]: Yeah maybe two cells so that copy to clipboard will work on each one.

**Review by ledell — CHANGES_REQUESTED** [review]

**Review by michalkurka — APPROVED** [review]
Looks good to me!

Please check the download website on nightly as soon as this is merged

File: h2o-dist/index.html [hunk 2]
```diff
@@ -547,6 +547,11 @@ <h2>Use H<sub>2</sub>O directly from Python</h2>
                     pip install tabulate<br/>
                     pip install future
                 </p>
+                <p> Optional: install `matplotlib` (required for plotting in Python):</p>
+                  <button class="btn copy_button" id="btnCopy8" data-clipboard-target='#to_copy8'></button>
```
Comment (michalkurka) [inline h2o-dist/index.html:551]: I think the id cannot be duplicate (#to_copy8) is already used

**Pull request merged**